Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'rails-controller-testing', '~> 0.1'
 gem 'govuk_ab_testing', '~> 2.0'
 gem 'htmlentities', '4.3.4'
 gem 'statsd-ruby', '1.3.0', require: 'statsd'
+gem 'sprockets-rails'
```

Comment:
Although SRI was introduced in 3.1.0 so there's no need to bump it up to 3.2. I'll remove it instead.